lib/systems/default.nix

```diff
@@ -90,6 +91,10 @@ rec {
           powerpcle = "ppc";
           powerpc64 = "ppc64";
           powerpc64le = "ppc64le";
+          mips = "mips";
```

Comment:
I think this is redundant - this attr set is just used to rewrite architectures when they are not the same in qemu. When they are the same, it is not necessary to include as the "or" clause covers this.